omero_search_engine/validation/results_validator.py

```diff
@@ -381,42 +528,43 @@ def get_containers_test_cases(self):
             search_omero_app.logger.info(mes)
         return mess
 
-    def compare_results(self):
+    def compare_results(self, operator=None):
         """
-        call the results
+        Get and compare the results from the postgresql and the searchengine
         """
         st_time = datetime.now()
-        self.get_results_postgres()
+        self.get_results_postgres(operator)
```

Comment:
maybe the method should be renamed
``get_results_from_db`` and ``get_results_from_search_engine``